.cargo/config

```diff
@@ -1,24 +1,46 @@
 [unstable]
 build-std = ["core"]
-configurable-env = true
+build-std-features = ["compiler-builtins-mem"]
 
-[target.thumbv6m-none-eabi]
-rustflags = [ 
-  "-C", "relocation-model=ropi",
-  "-C", "link-arg=-Tscript.ld",
-  "-C", "opt-level=3",
+# link.ld and device_layout.ld are picked up from nano-sdk
+[target.nanosplus]
+rustflags = [
   "-C", "passes=ledger-ropi",
   "-C", "codegen-units=1",
-  "-C", "lto",
+  "-C", "save-temps", # Fixes incremental builds by keeping .bc and .ll files
   "-C", "embed-bitcode",
   "--cfg", "target_os=\"nanos\"",
+  "--cfg", "target_device=\"nanosplus\"",
+  "-Z", "new-llvm-pass-manager=no",
   "--emit=obj,llvm-ir,llvm-bc"
 ]
-linker = "armv6m-unknown-none-eabi-clang"
-runner = "./speculos-wrapper -k 2.0"
+runner = "./speculos-wrapper -m nanosp "
 
-[build]
-target = "thumbv6m-none-eabi"
+[target.nanos]
+rustflags = [
+  "-C", "passes=ledger-ropi",
+  "-C", "codegen-units=1",
+  "-C", "save-temps", # Fixes incremental builds by keeping .bc and .ll files
+  "-C", "embed-bitcode",
+  "--cfg", "target_os=\"nanos\"",
+  "--cfg", "target_device=\"nanosplus\"",
```

Comment:
?

Reply:
This could be removed, this is from the previous commit I did, and nano_sdk branch used now does not make use of this.

Reply:
Ok